src/methods/modular_programs/hard/q7/RevUp.java

```diff
@@ -0,0 +1,31 @@
+package methods.modular_programs.hard.q7;
+
+/*
+Write a Java Program with a method called revString that accepts a string and returns it in reverse. Write a main method
```

Comment:
Change to:
Write a Java Program that asks the user to enter a sentence and calls a method called revString. The method accepts the string and returns it in reverse, which the main method then prints.